style.css

```diff
@@ -1,6 +1,178 @@
-@media (min-width:320px) { /* smartphones, iPhone, portrait 480x320 phones */ }
-@media (min-width:481px) { /* portrait e-readers (Nook/Kindle), smaller tablets @ 600 or @ 640 wide. */ }
-@media (min-width:641px) { /* portrait tablets, portrait iPad, landscape e-readers, landscape 800x480 or 854x480 phones */ }
-@media (min-width:961px) { /* tablet, landscape iPad, lo-res laptops ands desktops */ }
-@media (min-width:1025px) { /* big landscape tablets, laptops, and desktops */ }
-@media (min-width:1281px) { /* hi-res laptops and desktops */ }
\ No newline at end of file
+* {
+  padding: 0;
+  margin: 0;
+}
+body, html {
+  overflow-x: hidden;
+}
+nav {
+  margin: 0 auto;
+  max-width: 1100px;
+  width: 100%;
+}
+
+#checkbox-menu,
+.toggle {
+  display: none;
+}
+
+.menu {
+  align-items: center;
+  display: flex;
+  padding: 0;
+  margin: 0;
+  height: 50px;  
+  flex-direction: row;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/media_adaptive/pullRequest?prid=5498347)